sdk/nodejs/runtime/settings.ts

```diff
@@ -248,6 +249,7 @@ export async function awaitFeatureSupport(): Promise<void> {
         store.supportsAliasSpecs = await monitorSupportsFeature(monitorRef, "aliasSpecs");
         store.supportsTransforms = await monitorSupportsFeature(monitorRef, "transforms");
         store.supportsInvokeTransforms = await monitorSupportsFeature(monitorRef, "invokeTransforms");
+        store.supportsPackageReferences = await monitorSupportsFeature(monitorRef, "packageReferences");
```

Comment:
General thought -- could we use `Promise.all` here? Maybe that's a bad idea in terms of load, or its failure semantics, though I kinda feel like it's OK since if any of these fail now they throw anyway.

Reply:
I'll make a note and do it an another PR